external/llvm-project/mlir/lib/ExecutionEngine/RocmRuntimeWrappers.cpp

```diff
@@ -202,7 +196,116 @@ mgpuMemGetDeviceMemRef1dInt32(int32_t *allocated, int32_t *aligned,
   return {devicePtr, devicePtr, offset, {size}, {stride}};
 }
 
-extern "C" void mgpuSetDefaultDevice(int32_t device) {
```

Comment:
For example, setDefaultDevice() is used by the lowering of `gpu.set_default_device`